controllers/events.js

```diff
@@ -193,11 +200,77 @@ const endActiveEvent = async (req, res) => {
   }
 };
 
+/**
+ * Adds a peer to an event.
+ *
+ * @async
+ * @function
+ * @param {Object} req - The Express request object.
+ * @param {Object} res - The Express response object.
+ * @returns {Promise<Object>} The JSON response with the added peer data and a success message.
+ * @returns {Promise<Object>} The JSON response with an error message if an error occurred while adding the peer.
+ */
+const addPeerToEvent = async (req, res) => {
+  try {
+    const data = await eventQuery.addPeerToEvent({
+      peerId: req.body.peerId,
+      name: req.body.name,
+      role: req.body.role,
+      joinedAt: req.body.joinedAt,
+      eventId: req.params.id,
+    });
+    return res.status(200).json({
+      data,
+      message: SUCCESS_MESSAGES.CONTROLLERS.ADD_PEER_TO_EVENT,
+    });
+  } catch (error) {
+    logger.error({ error });
+    return res.status(500).json({
+      error: error.code,
+      message: ERROR_MESSAGES.CONTROLLERS.ADD_PEER_TO_EVENT,
+    });
+  }
+};
+
+/**
+ * Kicks out a peer from an event.
+ *
+ * @async
+ * @function
+ * @param {Object} req - The Express request object.
+ * @param {Object} res - The Express response object.
+ * @returns {Promise<Object>} The JSON response with a success message if the peer is successfully kicked out.
+ * @returns {Promise<Object>} The JSON response with an error message if an error occurred while kicking out the peer.
+ */
+const kickoutPeer = async (req, res) => {
+  const { id } = req.params;
+  const payload = {
+    peer_id: req.body.peerId,
+    reason: req.body.reason,
+  };
+
+  try {
+    await apiService.post(`/active-rooms/${id}/remove-peers`, payload);
```

Comment:
I will suggest to keep all the strings in constant 
1. easier to maintain
2. easier to replace
3. you remember constant names more often than long strings so they are easier to navigate and search

Reply:
Thanks a lot for your review, @vvaibhavdesai.
I have made all the strings as constant.